### DISCUSSION TITLE
[Request] ChurchCRM

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#1810] [General]
Requesting the following for support

https://github.com/ChurchCRM/CRM

Requires PHP, MYSQL to be installed prior to installation. 

Thanks in advance.

**Comment by tteck** [comment]
I know it's not ChurchCRM, but Odoo CRM is available in TurnKey.

**Reply by Dvalin21** [reply]
Thanks for the suggestion. I will have to check this out!
